.github/workflows/test.yml

```diff
@@ -21,8 +24,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install hatch
-          pip install .
+          pip install hatch "coverage[toml]"
```

Comment:
`pytest-cov` requires `coverage.py` to be installed globally. See the silently failing logs in the previous PRs ([silent failure](https://github.com/FaradayInstitution/BPX/actions/runs/12437661489/job/34728010041) in #84; [silent failure](https://github.com/FaradayInstitution/BPX/actions/runs/12436303784/job/34723880823) in #83).

Also see how codecov finally posts a coverage comment on this PR.